frontend/public/components/factory/list-page.jsx

```diff
@@ -68,15 +72,17 @@ export class ListPageWrapper_ extends React.PureComponent {
       rowFilters,
     } = this.props;
     const data = flatten ? flatten(this.props.resources) : [];
+    const filteredData = this.getFilteredRows(this.props.filters.name, data);
```

Comment:
We should try to find a way that avoids filtering the data twice. It's expensive for large lists, and the filter will run again after each character is typed.